test/index.test.js

```diff
@@ -102,166 +105,252 @@ describe('AdRoll', function() {
         analytics.stub(window.__adroll, 'record_user');
       });
 
-      it('should track page view with fullName', function() {
-        analytics.page('Category', 'Name', { url: 'http://localhost:34448/test/' });
-        analytics.called(window.__adroll.record_user, {
-        adroll_segments: 'viewed_category_name_page',
-        adroll_custom_data: {
-          path: '/test/',
-          referrer: '',
-          title: 'integrations tests',
-          search: '',
-          name: 'Name',
-          category: 'Category',
-          url: 'http://localhost:34448/test/'
-        }
+      describe('V1', function() {
+        beforeEach(function() {
+          adroll.options.v1 = true;
         });
-      });
 
-      it('should track unnamed/categorized page', function() {
-        analytics.page({ url: 'http://localhost:34448/test/' });
-        analytics.called(window.__adroll.record_user, {
-          adroll_segments: 'loaded_a_page',
-          adroll_custom_data: {
+        it('should track page view with fullName', function() {
+          analytics.page('Category', 'Name', { url: 'http://localhost:34448/test/' });
+          analytics.called(window.__adroll.record_user, {
+            adroll_segments: 'viewed_category_name_page',
             path: '/test/',
             referrer: '',
             title: 'integrations tests',
             search: '',
+            name: 'Name',
+            category: 'Category',
             url: 'http://localhost:34448/test/'
-          }
+          });
+        });
+
+        it('should track unnamed/categorized page', function() {
+          analytics.page({ url: 'http://localhost:34448/test/' });
+          analytics.called(window.__adroll.record_user, {
+            adroll_segments: 'loaded_a_page',
+            path: '/test/',
+            referrer: '',
+            title: 'integrations tests',
+            search: '',
+            url: 'http://localhost:34448/test/'
```

Comment:
I realize that you're following precedent on these tests, but can you (let's do it in a separate PR) change these tests to not have hardcoded URLs/ports? Hardcoding these means that if we ever switch to a different test runner, change Duo's port number, run these tests in a different environment, use Mocha's `grep` querystring to whittle down tests. etc. etc. they'll all fail.